plugin/trino-memsql/pom.xml

```diff
@@ -58,6 +58,12 @@
             <artifactId>guice</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.singlestore</groupId>
+            <artifactId>singlestore-jdbc-client</artifactId>
+            <version>1.0.1</version>
```

Comment:
Is LGPL-2.1 a no-go, @martint ?

Reply:
I was going to ask if a new dependency needed to be vetted or documented somewhere (in our list of open source licenses?).
Btw, the MariaDB driver this is replacing is also LGPL-2.1:
( https://search.maven.org/artifact/org.mariadb.jdbc/mariadb-java-client/2.7.2/jar )
The SingleSource driver is forked from it.

Reply:
> Btw, the MariaDB driver this is replacing is also LGPL-2.1:

thanks for pointing this out.

I recall some offline discussion with the conclusion that LGPL dependency is OK, but still would prefer @martint  to confirm. Will keep 'request for changes' for now here.

Reply:
Thanks for raising it - I have this as a draft as I'm completely new to the process (and the codebase), so all feedback and issues are great for me to be aware of 👍